### PR TITLE
Add a report window if there were errors during the renaming process

### PR DESCRIPTION
**Pull request by janus2**
* Need Humdinger touch for the wording.

### CODE REVIEW

File: Main/ReportWindow.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+				PreviewField* previewField = new PreviewField(listItem->fNewName);
+				previewField->SetError(true);
+				int32 i = 0;
+				row->SetField(new BStringField(listItem->fName), i++);
+				row->SetField(previewField, i++);
+				BString errorMsg;
+				switch (listItem->Error()) {
+					case 0 : // this should never happen
+						break; 
+					case 1 : errorMsg = B_TRANSLATE("Duplicated name");
```
Comment (humdingerb) [inline Main/ReportWindow.cpp]: Why is there a different message for case 1 and case 2?

**Comment by humdingerb** [comment]
In the report window, maybe we can do away with the BBox and just add the heading with a BStringView.

**Comment by humdingerb** [comment]
Shouldn't the report window only appear, if the conflicts popped up uexpectedly, i.e. NOT oif they've been annonced beforehand via the red Preview hightlighing?

File: Main/ReportWindow.cpp (hunk 1 above)
Comment (janus2) [reply Main/ReportWindow.cpp]: we spoke about this alot of time
1) is the duplicated already discovered by the application
2) this error happen when the application finds a duplicate on disk

**Comment by janus2** [comment]
You are a difficult customer;-) 
> In the report window, maybe we can do away with the BBox and just add the heading with a BStringView.

It's the first thing I do but doesn't look very nice, I left the code there is a simple modification.

>Shouldn't the report window only appear, if the conflicts popped up uexpectedly, i.e. NOT oif they've been annonced beforehand via the red Preview hightlighing?

I don't now is your idea!!!

File: Main/ReportWindow.cpp (hunk 1 above)
Comment (humdingerb) [reply Main/ReportWindow.cpp]: OK. But why is there a distinction if the reason is for both "File with that name already exists"?

**Comment by humdingerb** [comment]
> It's the first thing I do but doesn't look very nice, I left the code there is a simple modification.

It should look like Haiku's Repositories preferences, right? That doesn't look bad to me...

> > Shouldn't the report window only appear, if the conflicts popped up uexpectedly, i.e. NOT oif they've been annonced beforehand via the red Preview hightlighing?
> I don't now its your idea!!!

I thought we mentioned that before... All I find now is this last comment of #32 (back then still as alert instead of report)
We've spread the conversation out to too many PRs, issues, etc. :)

**Comment by janus2** [comment]
Fixed in PR #35

**Pull request merged**